tensorboard/plugins/interactive_inference/README.md

```diff
@@ -316,3 +316,15 @@ For TensorFlow GPU support, use the witwidget-gpu package instead of witwidget.
 
 Then, use it as seen at the bottom of the
 [What_If_Tool_Notebook_Usage.ipynb notebook](https://colab.research.google.com/github/tensorflow/tensorboard/blob/master/tensorboard/plugins/interactive_inference/What_If_Tool_Notebook_Usage.ipynb).
+
+### How do I enable it for use in a JupyterLab or Cloud AI Platform notebook?
+Install and enable WIT for JupyterLab by running a cell containing:
+```
+!pip install witwidget
+!jupyter labextension install wit-widget
+!jupyter labextension install @jupyter-widgets/jupyterlab-manager
+```
+For TensorFlow GPU support, use the witwidget-gpu package instead of witwidget.
```

Comment:
nit: Code spans around `witwidget-gpu`, `witwidget`, and the `!sudo`
command on the next line.

Reply:
done